docs/src/components/menu/menu.module.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../pages/global';
+
+.desktopMenu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: var(--sidebar-width);
+  overflow-y: auto;
+  @include respond-to('mobile') {
+    display: none;
+  }
+}
+
+.mobileMenu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: var(--topbar-height);
+  display: flex;
+  @include respond-to('desktop') {
+    display: none;
+  }
+}
+
+.mobileMenu,
+.desktopMenu {
+  background-color: $menuBackgroundColor;
+  color: $menuTextColor;
+  padding: 20px;
```

Comment:
As a different comment, I think we typically want to use 'rem' values instead of px values for things like this (at least with padding and margins)

Reply:
Hmmm... I'm not sure we have a standard; the Olive homepage uses `px` all over the place. We should standardize on something though.